style.css

```diff
@@ -0,0 +1,201 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap");
+
+/* Common styles */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border - box;
+  text-decoration: none;
+  list-style: none;
+  outline: none;
+  font-family: "Inter", 'Courier New', Courier, monospace;
+  font-weight: 400;
+}
+
+html {
+  font-size: 62.5%;
+
+  --white: #fff;
+  --buttons-color: #36b37f;
+  --Primary-text-color: #172b4d;
+  --Primary-text-color2: #3a4a42;
+  --lighterG1: #c5e9da;
+  --lighterG2: #ebf0ee;
+  --Pressed: #008552;
+}
+
+body {
+  width: 100%;
+  height: 100vh;
+}
+
+.main-wrapper {
+  width: 100%;
+  height: 100vh;
+}
+
+/* End of Common Styles */
+
+@media (max-width: 750px) {
+  /* Navbar */
+
+  .navbar {
+    display: block;
+    width: 50%;
+    height: 2.75rem;
+    margin: 20px 12px;
+  }
+
+  .Hamburger {
+    position: absolute;
+    cursor: pointer;
+    width: 18px;
+    height: 1.2rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    right: 36.612px;
+    top: 40px;
+
+    /* border: red solid 3px;  */
```

Comment:
- [ ] Kindly remove all commented out and unwanted pieces of code from your code base. It makes it look messy and cluttered.